obstacle_range: 2.5
raytrace_range: 3.0
# Modify with actual dimensions of the robot
footprint: [[-0.325, -0.25], [-0.325, 0.25], [0.325, 0.25], [0.325, -0.25]]
inflation_radius: 0.3

observation_sources: laser_scan_sensor 
laser_scan_sensor: {sensor_frame: lidar, 
                    data_type: LaserScan, 
                    topic: /lidar/ground_truth, 
                    marking: true, 
                    clearing: true}